### PR TITLE
Fix for SFPF Clear CACHE_INFO

### PR DESCRIPTION
**Pull request by habrahams**
Fix for #3868

### CODE REVIEW

File: src/objects/zcl_abapgit_object_sfpf.clas.abap [hunk 1]
```diff
@@ -309,12 +309,33 @@ CLASS ZCL_ABAPGIT_OBJECT_SFPF IMPLEMENTATION.
...
+          li_node_iter       TYPE REF TO if_ixml_node_iterator,
+          li_node            TYPE REF TO if_ixml_node,
+          li_node_new        TYPE REF TO if_ixml_node,
+          li_node_parent     TYPE REF TO if_ixml_node.
 
     lv_xstr = form_to_xstring( ).
     li_document = cl_ixml_80_20=>parse_to_document( stream_xstring = lv_xstr ).
+
+*   Clear CACHE_INFO
+    li_node_collection = li_document->get_elements_by_tag_name_ns('CACHE_INFO').
```
Comment (larshp) [inline src/objects/zcl_abapgit_object_sfpf.clas.abap:324]: add spaces for the parameters, 

```suggestion
    li_node_collection = li_document->get_elements_by_tag_name_ns( 'CACHE_INFO' ).
```

File: src/objects/zcl_abapgit_object_sfpf.clas.abap [hunk 2]
```diff
@@ -309,12 +309,33 @@
...
+    IF li_node_collection IS NOT INITIAL.
+      li_node_iter = li_node_collection->create_iterator( ).
+      DO.
+        li_node = li_node_iter->get_next( ).
+        IF li_node IS INITIAL.
+          EXIT.
+        ENDIF.
+         li_node_new = li_document->create_element_ns('CACHE_INFO').
```
[the remark this answers is not in the document]
Comment (FreHu) [reply src/objects/zcl_abapgit_object_sfpf.clas.abap:332]: @larshp let's see if it catches the missing spaces before braces here once the indentation is fixed

File: src/objects/zcl_abapgit_object_sfpf.clas.abap (hunk 1 above)
Comment (habrahams) [reply src/objects/zcl_abapgit_object_sfpf.clas.abap]: Done

**Comment by habrahams** [comment]
@larshp , am I missing a step for this pull request to be merged?

**Comment by larshp** [comment]
no, I just would like to test it before merging, have not gotten around to it

now the PR is in draft mode? is it ready to be merged?

**Comment by habrahams** [comment]
Ah ok, thanks, sorry about that, I thought I was missing a step in the progress. I have marked it as ready for review.

**Comment by larshp** [comment]
thanks 👍

**Pull request merged**